modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/date_facet.js

```diff
@@ -15,18 +15,20 @@ AUI.add(
 			const instance = this;
 
 			instance.form = config.form;
-			instance.fromInputDatePicker = config.fromInputDatePicker;
 			instance.fromInputName = config.fromInputName;
 			instance.namespace = config.namespace;
 			instance.parameterName = config.parameterName;
 			instance.searchCustomRangeButton = config.searchCustomRangeButton;
 			instance.searchCustomRangeToggleName =
 				config.searchCustomRangeToggleName;
-			instance.toInputDatePicker = config.toInputDatePicker;
 			instance.toInputName = config.toInputName;
 
-			instance.fromInput = A.one('#' + instance.fromInputName);
-			instance.toInput = A.one('#' + instance.toInputName);
+			instance.fromInputDatePicker = this._getInputDatePicker(
+				config.fromInputName
+			);
+			instance.toInputDatePicker = this._getInputDatePicker(
+				config.toInputName
+			);
```

Comment:
Should this be `instance._getInputDatePicker` instead of `this`?

Reply:
Oops, thank you for catching that!